src/apps/LandingRegister/Header.js

```diff
@@ -4,55 +4,88 @@ import {IconButton} from 'material-ui';
 
 export default Radium(() => {
   const styles = {
-    headerDiv: {
+    header: {
+      marginTop: 21,
+      marginBottom: 71,
+      width: '100%',
+      display: 'flex',
       justifyContent: 'flex-end',
-      display: 'flex'
+      '@media (max-width: 750px)': {
+        justifyContent: 'space-between',
+        marginBottom: 40
+      }
+    },
+    headerLogo: {
+      display: 'none',
+      '@media (max-width: 750px)': {
+        display: 'flex',
+        paddingLeft: 16
+      }
+    },
+    headerLogoImg: {
+      width: 175,
+      height: 37,
```

Comment:
It doesn't because the Chrome but because image is contained by `div` with `display: flex`